src/LinuxTracingIntegrationTests/LinuxTracingIntegrationTest.cpp

```diff
@@ -316,6 +316,7 @@ class LinuxTracingIntegrationTestFixture {
     capture_options.set_trace_context_switches(true);
     capture_options.set_pid(puppet_.GetChildPid());
     capture_options.set_samples_per_second(1000.0);
+    capture_options.set_stack_dump_size(65000);
```

Comment:
Move this down one line to follow the definitions in the proto.

Reply:
I moved the line in the proto instead. I think it is a bit better, as this option is independent of the unwinding method.